task7/index.js

```diff
@@ -0,0 +1,34 @@
+const div = document.getElementById("js-loading");
+const gif = document.createElement('img');
+gif.src = "loading-circle.gif";
+div.appendChild(gif);
```

Comment:
Make it a function and run it in the appropriate place.　I was pointed out the same thing.

Reply:
Thank you for your comment.It was very helpful.
I'm worried that my lack of understanding has made the corrections appropriate for the comments.

Reply:
I think that's good. 
(nits)If you rewrite the following HTML, you can put more elements inside the function.
`    <div id="js-loading">
      <ul id ="js-ul"></ul>
    </div>`
` <div id="js-loading"></div>
      <ul id ="js-ul"></ul>
    `

Reply:
I corrected it based on the comments I received. Thank you!